### PR TITLE
Valgrind News

### PR DESCRIPTION
**Pull request by paulfloyd**

### CODE REVIEW

**Review by pauamma — COMMENTED** [review]
One sentence per line, but can be fixed later in the editorial pipeline unless you have other changes coming. Otherwise LGTM.

**Comment by paulfloyd** [comment]
> One sentence per line, but can be fixed later in the editorial pipeline unless you have other changes coming. Otherwise LGTM.

Just the valgrind-devel version and date when the patch lands.

**Review by lsalvadore — COMMENTED** [review]
I have made a small suggestion for the title.

I also suggest using some kind of special fonts for function names and similar things to improve readability (e.g. `_memcheck_`), but if you want to keep it as it is it is fine.

File: website/content/en/status/report-2023-07-2023-09/valgrind.adoc [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+Such usage is not portable and is deprecated (C23 will make it Undefined Behaviour). memcheck
+now validates the values used for alignment and sized delete for memalign, posix_memalign, aligned_alloc
+and all aligned and sized overloads of operator new and operator delete. Reading DWARF debuginfo is
+now done in a lazy manner which can improve performance.
+
+As usual there are numerous small bugfixes.
+
+Specific to FreeBSD there is support for FreeBSD 15. Two extra _umtx_op operations are now supported,
+UMTX_OP_GET_MIN_TIMEOUT and UMTX_OP_SET_MIN_TIMEOUT. There is a fix for the use of
+sysctl kern proc pathname with the guest pid or -1 which previously returned the path of the Valgrind host.
```
Comment (grahamperrin) [inline website/content/en/status/report-2023-07-2023-09/valgrind.adoc:22]:
```suggestion
Specific to FreeBSD there is support for FreeBSD 15. Two extra `_umtx_op` operations are now supported, `UMTX_OP_GET_MIN_TIMEOUT` and `UMTX_OP_SET_MIN_TIMEOUT`.
There is a fix for the use of sysctl `kern.proc.pathname` with the guest pid or -1, which previously returned the path of the Valgrind host.
```
Was _kern proc pathname_ (with spaces) intended?

```text
% sysctl -d kern.proc.pathname
kern.proc.pathname: Process executable path
% 
```

Whilst here: a comma.

**Review by grahamperrin — CHANGES_REQUESTED** [review]

File: website/content/en/status/report-2023-07-2023-09/valgrind.adoc (hunk 1 above)
Comment (paulfloyd) [reply website/content/en/status/report-2023-07-2023-09/valgrind.adoc:22]: I'd rather leave it as it is. It's not a command line sysctl. Strictly speaking not even a libc sysctl.

**Comment by lsalvadore** [comment]
Merged with commit 6a6e4c93d28c34618c0b3cb487c8a82636720c22, thanks.

**Pull request closed without merging**